cpp/src/copying/gather.cu

```diff
@@ -115,7 +101,12 @@ std::unique_ptr<table> gather(table_view const& source_table,
                               rmm::mr::device_memory_resource* mr)
```

Comment:
is `check_bounds`  okay to be `bool`?

Reply:
I talked with @harrism and seeing as it is a single `bool` (vs 3 back-to-back) and it is a non-`detail` `namespace` API, he agreed we should leave it.